pages/common/streamlink.md

```diff
@@ -0,0 +1,36 @@
+# Streamlink
+
+> Extracts streams from various services and pipes them into a video player of choice.
+> More information: <https://streamlink.github.io>.
+
+- Attempt to extract streams from the URL specified, and if it's successful, print out a list of available streams to choose from:
+
+`streamlink {{example.com/stream}}`
+
+- Open a stream with specified quality:
+
+`streamlink {{example.com/stream}} {{720p60}}`
+
+- Use **best** or **worst** for selecting the highest or lowest available quality:
+
+`streamlink {{example.com/stream}} {{best}}`
+
+- Specify which player to use with --player to feed stream data to. By default, VLC will be used if it can be found in its default location:
+
+`streamlink --player=mpv {{example.com/stream}} {{best}}`
+
+- Specify the amount of time to skip from the beginning of the stream. For live streams, this is a negative offset from the end of the stream (rewind):
+
+`streamlink --hls-start-offset {{[HH:]MM:SS}} {{example.com/stream}} {{best}}`
+
+- Skip to the beginning of a live stream, or as far back as possible:
+
+`streamlink --hls-live-restart {{example.com/stream}} {{best}}`
+
+- Write stream data to a file instead of playing it:
+
+`streamlink --output {{path/to/file}} {{example.com/stream}} {{best}}`
```

Comment:
A minor thing, but is there a default format it writes in? If so, we can reflect that with `path/to/file.mp4` for example if the default was mp4.

Reply:
It seems that it writes in MPEG transport stream, so I'll add the **.ts** extension.